node/test/performance-test/Cargo.toml

```diff
@@ -10,11 +10,14 @@ quote = "1.0.26"
 env_logger = "0.9"
 log = "0.4"
 
-polkadot-node-core-pvf = { path = "../../core/pvf" }
+polkadot-node-core-pvf-worker = { path = "../../core/pvf/worker" }
 polkadot-erasure-coding = { path = "../../../erasure-coding" }
 polkadot-node-primitives = { path = "../../primitives" }
 polkadot-primitives = { path = "../../../primitives" }
 
+sc-executor-common = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sp-maybe-compressed-blob = { git = "https://github.com/paritytech/substrate", branch = "master" }
```

Comment:
Makes sense for the worker crate to `pub use` these two.

Reply:
Can you explain the reasoning? Seems to me like unnecessary indirection.

Reply:
Well, if every crate that uses `worker` also needs these 2, we might as well just re-export them. A matter of taste so feel free to disregard.